src/CTA.Rules.Update/ActionsRewriter.cs

```diff
@@ -321,11 +321,12 @@ public override SyntaxNode VisitCompilationUnit(CompilationUnitSyntax node)
         }
         public override SyntaxNode VisitObjectCreationExpression(ObjectCreationExpressionSyntax node)
         {
-            ObjectCreationExpressionSyntax newNode = node;// (ObjectCreationExpressionSyntax)base.VisitObjectCreationExpression(node);
-            bool skipChildren = false;
+            var symbols = _semanticModel.GetSymbolInfo(node);
+            ExpressionSyntax newNode = node;// (ObjectCreationExpressionSyntax)base.VisitObjectCreationExpression(node);
```

Comment:
Can we remove the commented code here?

Reply:
Yep, I left it there only for context of what was there before but the CR file diff achieves that perfectly on its own. Removed as suggested.